saleor/attribute/migrations/0031_assignedpageattributevalue_page.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.20 on 2023-08-24 11:25
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.db.models.signals import post_migrate
+from django.apps import apps as registry
+
+from .tasks.saleor3_16 import assign_pages_to_attribute_values_task
+
+
+def data_migration(apps, _schema_editor):
```

Comment:
It's good idea to separate data migration with schema changes.

Reply:
done